Hardware/Nic/NicGroup.cs

```diff
@@ -1,34 +1,50 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using System.Linq;
 using System.Net.NetworkInformation;
 using System.Text;
 
 namespace OpenHardwareMonitor.Hardware.Nic
 {
     internal class NicGroup : IGroup
     {
-
-        private List<Hardware> hardware = new List<Hardware>();
-        private NetworkInterface[] _networkInterfaces;
+        private readonly ISettings _settings;
+        private List<Nic> _hardware = new List<Nic>();
 
         public NicGroup(ISettings settings)
         {
-            int p = (int)Environment.OSVersion.Platform;
-            if ((p == 4) || (p == 128))
-            {
-                hardware = new List<Hardware>();
+            _settings = settings;
+            if (Software.OperatingSystem.IsLinux)
                 return;
```

Comment:
Not sure why this was added in the first place, but NetworkInterface is available on Linux as well, so this check can be removed.